python/py-yarl/Portfile

```diff
@@ -27,8 +28,8 @@ checksums           rmd160  b0f0a7cdf55a7188de0a962d8393f895c234b517 \
 
 if {${name} ne ${subport}} {
     depends_build       port:py${python.version}-setuptools
-    depends_lib         port:py${python.version}-idna
-    depends_lib         port:py${python.version}-multidict
+    depends_lib         port:py${python.version}-idna \
+                        port:py${python.version}-multidict
```

Comment:
While you're on it, could you change `depends_build` and `depends_lib` to `depends_build-append` and `depends_lib-append`?